src/notebookConverter.ts

```diff
@@ -1,40 +1,44 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import * as os from 'os';
+import * as os from "os";
 import {
-    CodeAction,
-    CodeActionContext,
-    CodeLens,
-    Command,
-    CompletionItem,
-    CompletionList,
-    Diagnostic,
-    DiagnosticRelatedInformation,
-    Disposable,
-    DocumentHighlight,
-    DocumentLink,
-    DocumentSymbol,
-    Event,
-    EventEmitter,
-    Hover,
-    Location,
-    LocationLink,
-    Position,
-    Range,
-    SymbolInformation,
-    TextDocument,
-    TextDocumentChangeEvent,
-    TextDocumentContentChangeEvent,
-    TextEdit,
-    Uri,
-    WorkspaceEdit,
-    NotebookDocument
-} from 'vscode';
-import { IVSCodeNotebook } from './common/types';
-import { InteractiveInputScheme, InteractiveScheme, NotebookCellScheme } from './common/utils';
-import { IFileSystem } from './common/types';
-import { IConcatTextDocument } from './concatTextDocument';
-import { NotebookConcatDocument } from './notebookConcatDocument';
+  CodeAction,
```

Comment:
did the indentation change from 4 spaces to 2 spaces?

Reply:
Not sure what's doing that? Prettier is my default formatter. Looks like it did? I'll add a .prettierrc.js